airflow/helm/airflow/values.yaml.tpl

```diff
@@ -22,7 +24,9 @@ sshConfig:
   id_rsa_pub: example
 {{ end }}
 
+{{ if .Values.postgresqlEnabled }}
```

Comment:
like with redis and gitsync, this should default on, so we'd need something like:

{{ if not .Values.postgresDisabled }}